src/ucp/rma/put_offload.c

```diff
@@ -14,8 +14,68 @@
 #include <ucp/core/ucp_request.inl>
 #include <ucp/dt/datatype_iter.inl>
 #include <ucp/proto/proto_multi.inl>
+#include <ucp/proto/proto_single.inl>
 
 
+static ucs_status_t ucp_proto_put_offload_short_progress(uct_pending_req_t *self)
+{
+    ucp_request_t                   *req = ucs_container_of(self, ucp_request_t,
```

Comment:
[minor] unusual code alignment used for variable declaration
below in the PR changes, you don't use such alignment